core/jobs/batch_jobs/verify_exploration_migration_job.py

```diff
@@ -0,0 +1,192 @@
+            android_proto_size_in_bytes field population.
+        """
+        exploration_models = (
+            self.pipeline
+            | 'Get all non-deleted exploration models' >> (
+                ndb_io.GetModels(
+                    exp_models.ExplorationModel.get_all(
+                        include_deleted=False)
+                    )
+                )
+            | 'Remove the Exploration Temporary' >> beam.Filter(
+                lambda exp_model: exp_model.id != 'umPkwp0L1M0-')
+            # Pylint disable is needed because pylint is not able to correctly
+            # detect that the value is passed through the pipe.
+            | 'Add exploration keys' >> beam.WithKeys( # pylint: disable=no-value-for-parameter
+                lambda exp_model: exp_model.id)
+        )
+
+        exploration_results = (
+            exploration_models
+            | 'Transform and migrate model' >> beam.MapTuple(
+                self.fetch_exploration)
+        )
+        explorations_list = (
+            exploration_results
+            | 'Filter oks' >> beam.Filter(
+                lambda result_item: result_item.is_ok())
+            | 'Unwrap oks' >> beam.Map(
+                lambda result_item: result_item.unwrap())
+        )
+        exploration_job_run_results = (
+            exploration_results
+            | 'Generate results for migration' >> (
+                job_result_transforms.ResultsToJobRunResults(
+                    'EXPLORATION PROCESSED'))
+        )
+
+        exploration_object_list = (
+            {
+                'exploration': explorations_list,
+            }
+            | 'Merge object' >> beam.CoGroupByKey()
+            | 'Get rid ID' >> beam.Values()  # pylint: disable=no-value-for-parameter
+            | 'Remove unmigrated exploration object' >> beam.Filter(
+                lambda x: len(x['exploration']) > 0)
+            | 'Reorganize the exploration object' >> beam.Map(lambda objects: {
+                    'exploration': objects['exploration'][0]
+                })
+        )
+
+        exploration_boolean_list = (
+            exploration_object_list
+            | 'Generate exploration changes' >> beam.FlatMap(
+                lambda exp_objects: self.count_exploration(
+                    exp_objects['exploration']
+                ))
+        )
+
+        exploration_objects_list = (
+            {
+                'exploration_boolean_list': exploration_boolean_list
+            }
+            | 'Reorganize the exploration objects' >> beam.Map(lambda objects: {
+                    'exploration_boolean_list': objects['exploration_boolean_list'][0], # pylint: disable=line-too-long
+                })
+        )
+
+        exploration_objects_list_job_run_results = (
+            exploration_objects_list
+            | 'Transform exploration objects into job run results' >> (
+                job_result_transforms.CountObjectsToJobRunResult(
```

Comment:
Why don'ŧ you also use `ResultsToJobRunResults` here (I know that you may need to change some stuff above)?